src/BackButtonManager.Uno.WinUI/BackButtonManager.Uno.WinUI.csproj

```diff
@@ -11,9 +11,12 @@
     <Description>Chinook.BackButtonManager is a collection of extensible libraries to ease development around back button handling.</Description>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageReadmeFile>README.md</PackageReadmeFile>
-    <PackageTags>back;navigation;mvvm;maui;winui;net6.0-ios;net6.0-android;chinook</PackageTags>
```

Comment:
I think back and navigation as separate tags still make sense?

Reply:
I'll add them back